software/28.Gin/SETUP.md

```diff
@@ -0,0 +1,36 @@
+# Setup
+
+## 1. Go
+
+In this workshop, we will create an HTTP REST server using Go.
+First of all, make sure you have **Go 1.14** or higher installed, allowing you to use the Gin Web Framework.
+
+If you have the right version installed, you should have something similar:
+```shell
+go version
+> go version go1.18.2 linux/amd64 # OK
+```
+
+> The language should NOT be an issue in this workshop, if you're struggling with Go, do not hesitate to ask for help !
+
+## 2. Postman
+
+We will be using `postman` to test out our routes.
+
+- [Postman](https://www.postman.com/downloads/)
+
```

Comment:
There is an issue with postman, users should not install it with snap because its not updated on this package manager. You can precise it.